doc/source/about/index.rst

```diff
@@ -30,36 +30,30 @@ to the project.  For a list of those members and a description of their
 contributions to the code, see
 `our members website. <https://yt-project.org/members.html>`_
 
-For an up-to-date list of everyone who has contributed to the yt codebase,
-see the current `CREDITS <https://github.com/yt-project/yt/blob/main/CREDITS>`_ file.
-For a more detailed breakup of contributions made by individual users, see out
-`Open HUB page <https://www.openhub.net/p/yt_amr/contributors?query=&sort=commits>`_.
-
 History of yt
 -------------
 
-yt was originally begun by Matthew Turk in 2007 in the course of his graduate
-studies in computational astrophysics.  The code was developed
-as a simple data-reader and exporter for grid-based hydrodynamical simulation
-data outputs from the *Enzo* code.  Over the next few years, he invited
-collaborators and friends to contribute and use yt.  As the community grew,
-so did the capabilities of yt.  It is now a community-developed project with
-contributions from many people, the hospitality of several institutions, and
-benefiting from numerous grants.  With this community-driven approach
-and contributions from a sizeable population of developers, it has evolved
-into a fully-featured toolkit for analysis and visualization of
-multidimensional data.  It relies on no proprietary software -- although it
-can be and has been extended to interface with proprietary software and
-libraries -- and has been designed from the ground up to enable users to be
-as immersed in the data as they desire.
```

Comment:
I know you're very humble, Matt, but I think it is useful for people to know the whole history of yt started with you.  I like what you did below, but perhaps continuing to mention that it started in 2007 with your work on enzo analysis.

Reply:
Oh, now I see the thing below about you as the original author.  OK, that should be good enough.  But please, you need to get credit for having done an ENORMOUS amount on this code.

Reply:
If we can figure out a way to include it as part of a broader story, maybe in governance, so that it's not just About Matt, then I would be okay with it.

Reply:
I like stories About Matt, especially when they're true like this one. :)